binding.go

```diff
@@ -118,6 +119,45 @@ func JSON(model interface{}, opts ...Options) flamego.Handler {
 	})
 }
 
+// Yaml returns a middleware handler that injects a new instance of the model
+// with populated fields and binding.Errors for any deserialization, binding, or
+// validation errors into the request context. The model instance fields are
+// populated by deserializing the Yaml payload from the request body.
+func Yaml(model interface{}, opts ...Options) flamego.Handler {
```

Comment:
```suggestion
// YAML returns a middleware handler that injects a new instance of the model
// with populated fields and binding.Errors for any deserialization, binding, or
// validation errors into the request context. The model instance fields are
// populated by deserializing the YAML payload from the request body.
func YAML(model interface{}, opts ...Options) flamego.Handler {
```

Please also fix other occurrences.